src/hapi/pipelines/app/pipelines.py

```diff
@@ -29,6 +30,7 @@ def __init__(
         self.configuration = configuration
         self.session = session
         self.locations = Locations(configuration, session, use_live)
```

Comment:
I know this was done in a different PR, but should we limit locations (and subsequently admin regions) just to HRPs for now?

Reply:
I can make that change in a new PR

Reply:
Actually I'll do it in this one